docs/src/main/asciidoc/redis-reference.adoc

```diff
@@ -0,0 +1,105 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Redis Reference Guide
+
+:extension-status: preview
+include::./attributes.adoc[]
+:numbered:
+:sectnums:
+
+[[custom_redis_commands]]
+== How to use custom Redis Commands
+
+As the list of commands which are supported out-of-box by both https://github.com/quarkusio/quarkus/blob/main/extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/RedisClient.java[`RedisClient`] and https://github.com/quarkusio/quarkus/blob/main/extensions/redis-client/runtime/src/main/java/io/quarkus/redis/client/reactive/ReactiveRedisClient.java[`ReactiveRedisClient`] depends on what is available in https://github.com/vert-x3/vertx-redis-client[`vertx-redis-client`], then there might be case when you need a command which is not (yet) available via https://github.com/vert-x3/vertx-redis-client[`vertx-redis-client`].
+
+See:
+
+https://github.com/quarkusio/quarkus/issues/18351
+https://github.com/vert-x3/vertx-redis-client/issues/299
```

Comment:
Not sure it worths mentioning the issues here.

Reply:
I can remove these, as I wanted to mention what were the initial reasons why this was needed.

Reply:
Maybe as a note admonition?

Reply:
Yes, I think it will be better - as then someone could see the changes in a "bigger picture".

Reply:
Frankly, I would just drop them, they don't really bring much to the plate except noise.

Reply:
OK, I finally removed them.